octobot_trading/personal_data/orders/channel/orders.py

```diff
@@ -141,6 +159,38 @@ async def handle_post_open_orders_update(
             )
             await self._complete_open_order_init(waiting_complete_init_orders, is_from_bot)
 
+    async def _complete_pending_groups_from_storage(self, pending_groups):
+        # create order groups and their associated self-managed orders if any
+        # use copy.copy(pending_groups) as new groups might be added from new orders,
+        # in this case, also create the associated new orders
+        to_complete_groups = list(pending_groups.keys())
+        completed_groups = set()
+        max_allowed_nested_chained_orders_groups = 100
```

Comment:
I think we should create a constant somewhere to store this value

Reply:
I don't think so. I put it under a variable to add meaning instead of a magic number but it's something that will almost never happen, I don't think we should polute constants with it